modules/vue-plausible.yml

```diff
@@ -1,4 +1,4 @@
-name: plausible
+name: vue-plausible
```

Comment:
should we remove nuxt 3 compatibility here? the plugin format isn't wrapped in `defineNuxtPlugin` and may break in future

Reply:
I suggest to use `plausible-nuxt2` or `nuxt2-plausible` as new convention to specify legacy modules

Reply:
I am not comfortable with `nuxt2-plausible` since the author can always upgrade to Nuxt 3 at one point, where `vue-plausible` is actually the package name which is compatible with Nuxt.